fyde-app/README.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Fyde App
+has_children: false
+nav_order: 3
+permalink: /fyde-app
+---
+# Fyde App
+{: .no_toc }
+
+The Fyde app bundles the end user UI and the Fyde agent. It is available on the following platforms:
+
+* [iOS](https://www.fyde.com/downloads/ios){:target="_blank"}
+* [Android](https://www.fyde.com/downloads/android){:target="_blank"}
+* [MacOS](https://www.fyde.com/downloads/macos){:target="_blank"}
+* [Windows](https://www.fyde.com/downloads/windows){:target="_blank"}
+* [Linux](https://www.fyde.com/downloads/linux){:target="_blank"}
+
+## Fyde Agent
+
+The Fyde agent intersects requests at the network layer. When a DNS request matches a protected resource, it injects a reply, pointing the domain to an internally accessible marker IP address that represents the resource. It then, in parallel, initiates a request to the Access Policy Engine to grant permission to the resource, and establishes an mTLS connection to the Access Proxy that is associated with the resource.
```

Comment:
Maybe intercepts instead of intersects ?

Reply:
Nice catch.